app/frontend/src/lib/ui/components/filterGroup.scss

```diff
@@ -1,18 +1,41 @@
-.moj-filter {
+.filter-group__container {
+  border-bottom: 1px solid #b1b4b6;
```

Comment:
Is this a govuk-colour?

Reply:
no it isnt a gov color, i used the color that is the border for the tabs/table